op-chain-ops/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.21.1-alpine3.18 as builder
+
+RUN apk --no-cache add make
+
+COPY ./go.mod /app/go.mod
+COPY ./go.sum /app/go.sum
+
+WORKDIR /app
+
+RUN go mod download
+
+COPY ./op-service /app/op-service
+COPY ./op-node /app/op-node
+COPY ./op-plasma /app/op-plasma
+COPY ./op-chain-ops /app/op-chain-ops
+WORKDIR /app/op-chain-ops
+RUN make celo-migrate
+
+FROM alpine:3.18
+RUN apk --no-cache add ca-certificates bash rsync
+
+# RUN addgroup -S app && adduser -S app -G app
+# USER app
+WORKDIR /app
+
+COPY --from=builder /app/op-chain-ops/bin/celo-migrate /app
+ENV PATH="/app:${PATH}"
+
+ENTRYPOINT ["/app/celo-migrate-blocks"]
```

Comment:
Why `celo-migrate-blocks`?

Reply:
Oops, I forgot to change this in [e7a40cf](https://github.com/celo-org/optimism/pull/148/commits/e7a40cf277a65a6b55ca93626b7fb4176e66aa1a)